src/templates/partials/loadScript.html

```diff
@@ -3,9 +3,12 @@
   var site_base_path = '/'
   var algolia_projectId = '{{ algolia_projectId }}'
   var algolia_publicKey = '{{ algolia_publicKey }}'
-  var languages = '{{ languages }}'
+  var sdk_versions = '{{ sdk_versions }}'
```

Comment:
camelCase, please

(I try very hard not looking at the other 2 variables just above :astonished: )

Reply:
Also, missing semicolons